geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseIndexAdapter.scala

```diff
@@ -59,6 +59,10 @@ import org.opengis.feature.simple.{SimpleFeature, SimpleFeatureType}
 import scala.util.Random
 import scala.util.control.NonFatal
 
+import com.google.common.util.concurrent.ThreadFactoryBuilder
```

Comment:
could you use a non-guava class here instead? we have had a lot of issues with incompatible guava versions, so we try to avoid using it wherever possible.

Reply:
Yep, will fix.